code/ShareCare.php

```diff
@@ -197,8 +190,8 @@ public function EmailShareLink()
             return false;
         }
         $pageURL = $this->owner->AbsoluteLink();
-        $subject = rawurlencode('Thought you might like this');
-        $body = rawurlencode("Thought of you when I found this: $pageURL");
+        $subject = rawurlencode(_t('ShareCare.EmailSubject','Thought you might like this'));
+        $body = rawurlencode(_t('ShareCare.EmailBody','Thought of you when I found this: {URL}', [URL=>$pageURL]));
```

Comment:
This causes an error because URL should be wrapped in quotes. Can we go a step further and use PHP5.3 syntax as that's the minimum requirement for SS3.x. `array('URL' => $pageURL)`